### ISSUE TITLE
Which GPU mining algorithm can I use with 2 GB or less VRAM?

### ISSUE THREAD

**Issue by delphoir** [UnamSanctam/SilentCryptoMiner#2235]
When I try to use kawpow algorithm, the panel gives "not enough free VRAM" error with 2 GB of VRAM. any suggestions?

**Comment by UnamSanctam** [comment]
You would need to mine a coin that requires less VRAM, though if you only have 2GB of VRAM then you can't mine most coins https://minerstat.com/dag-size-calculator.

**Comment by delphoir** [comment]
Well, you're right but let's try and see. I'll update here

**Comment by delphoir** [comment]
Web panel shows miner status as "Stopped" instead of "Not enough VRAM"

**Comment by UnamSanctam** [comment]
It usually means that something is wrong with the pool configuration.

**Comment by delphoir** [comment]
Not sure what is wrong, wallet address is valid, I use kawpow algorithm and pool connection is: clore.2miners.com:12020

**Comment by UnamSanctam** [comment]
Can you post your configuration? It's also possible that your GPU is refusing to do something which could cause the "Stopped" status, it's possible for very low-end GPUs to do that.

**Comment by delphoir** [comment]
how do I find my configurations though?

**Comment by UnamSanctam** [comment]
You can run the checker on the computer where you ran the miner.

**Comment by delphoir** [comment]
Watchdog Running: Yes
Miners:
--cinit-algo=kawpow --pool=stratums://`ANHcdHjkMEYzSL6d8rc8JhvinRKeKJAemq`.{RANDOM}@clore.2miners.com:12020 --cinit-max-gpu=50 --cinit-stealth-targets="Taskmgr.exe,ProcessHacker.exe,perfmon.exe,procexp.exe,procexp64.exe,ModernWarfare.exe,ShooterGame.exe,ShooterGameServer.exe,ShooterGame_BE.exe,GenshinImpact.exe,FactoryGame.exe,Borderlands2.exe,EliteDangerous64.exe,PlanetCoaster.exe,Warframe.x64.exe,NMS.exe,RainbowSix.exe,RainbowSix_BE.exe,CK2game.exe,ck3.exe,stellaris.exe,arma3.exe,arma3_x64.exe,TslGame.exe,ffxiv.exe,ffxiv_dx11.exe,GTA5.exe,FortniteClient-Win64-Shipping.exe,r5apex.exe,VALORANT.exe,csgo.exe,PortalWars-Win64-Shipping.exe,FiveM.exe,left4dead2.exe,FIFA21.exe,BlackOpsColdWar.exe,EscapeFromTarkov.exe,TEKKEN 7.exe,SRTTR.exe,DeadByDaylight-Win64-Shipping.exe,PointBlank.exe,enlisted.exe,WorldOfTanks.exe,SoTGame.exe,FiveM_b2189_GTAProcess.exe,NarakaBladepoint.exe,re8.exe,Sonic Colors - Ultimate.exe,iw6sp64_ship.exe,RocketLeague.exe,Cyberpunk2077.exe,FiveM_GTAProcess.exe,RustClient.exe,Photoshop.exe,VideoEditorPlus.exe,AfterFX.exe,League of Legends.exe,Falluot4.exe,FarCry5.exe,RDR2.exe,Little_Nightmares_II_Enhanced-Win64-Shipping.exe,NBA2K22.exe,Borderlands3.exe,LeagueClientUx.exe,RogueCompany.exe,Tiger-Win64-Shipping.exe,WatchDogsLegion.exe,Phasmophobia.exe,VRChat.exe,NBA2K21.exe,NarakaBladepoint.exe,ForzaHorizon4.exe,acad.exe,AndroidEmulatorEn.exe,bf4.exe,zula.exe,Adobe Premiere Pro.exe,GenshinImpact.exe" --cinit-version="3.4.0" --cinit-idle-wait=5 --cinit-idle-gpu=90 --cinit-id="pmcaccbwscdupuqb"
GPUs:
 Intel(R) UHD Graphics
 NVIDIA GeForce MX110
Compatible GPU found: True

**Comment by UnamSanctam** [comment]
That looks correct, I assume it's your GPU that causes the miner to stop then.

**Comment by delphoir** [comment]
Yes, probably. Thanks for your help though. And I have one more question. What if I connect another GPU miner from another computer to the pool? Will it work together, or is it a lost cause? The other PC also has 2GB of VRAM with a relatively newer graphics card

**Comment by UnamSanctam** [comment]
They will work independently, but since it's a newer card it might work. Though due to the low VRAM it might not have enough.
